tests/mpool/test_mpool.sh

```diff
@@ -116,14 +116,14 @@ printf "sets\treps\tratio\n"
 
 # Test with full benefit from malloc pool (the binary begins with an
 # initial pool of 100).
-cmp_methods 10000 100 200 "full"
+cmp_methods 10000 100 180 "full"
 
 # mpool is still considerably faster than malloc in this range.
 cmp_methods 1000 1000 200 "partial"
```

Comment:
Should we change this one too?

Reply:
Not necessary; that's the "full benefit" from mpool.

freebsd 14 with clang
```sets	reps	ratio
10000	100	1.86
1000	1000	3.96
1	1000000	.88
```

freebsd14 with gcc
```
sets	reps	ratio
10000	100	2.45
1000	1000	5.02
1	1000000	.93
```

(repeated tests show little change in these numbers)

Reply:
Ok, I'm just confused by the word "partial".  I assumed that meant that we would see less of a benefit, but it seems like it's more?

Reply:
wait, my previous comment was garbage.  I need to investigate this some more.